src/data_readers/data_reader_jag_conduit.cpp

```diff
@@ -280,7 +326,11 @@ void data_reader_jag_conduit::set_all_scalar_choices() {
   conduit::NodeConstIterator itr = n_scalar.children();
```

Comment:
Consider altering to use:
const std::vector<std::string> &names = n_scalar.shild_names();
for (auto t : names) {
  ...
}
My reasoning is that data_store can supply something similar:
const std::vector<std::string> &names = m_data_store->child_names("0/outputs/scalars");